Dockerfile.rocm

```diff
@@ -25,8 +25,8 @@ ARG ARG_PYTORCH_ROCM_ARCH="gfx90a;gfx942"
 ENV PYTORCH_ROCM_ARCH=${ARG_PYTORCH_ROCM_ARCH}
 
 # Install some basic utilities
-RUN apt-get update && apt-get install python3 python3-pip -
-RUN apt-get update && apt-get install -y \
+RUN apt-get update -q -y --force-yes && apt-get install -q -y --force-yes python3 python3-pip -
```

Comment:
The trailing "-" was supposed to be "-y" actually

Reply:
hm... didn't know that :) will remove.